silo/management/commands/get_all_ona_forms.py

```diff
@@ -14,6 +16,8 @@ class Command(BaseCommand):
     """
     help = 'Fetches all reads that have autopull_frequency checked and belong to a silo'
 
+    logger = logging.getLogger("silo")
```

Comment:
I would recommend to use `__name__` instead of silo.

By doing that, you don't need to write `get_all_ona_forms` in every log. The variable __name__ is current module name in Python. For example, you call logger.getLogger(__name__) in a module "foo.bar.my_module", then it is logger.getLogger("foo.bar.my_module"). When you need to configure the logger, you can configure to “foo”, then all modules in "foo" packages shares same configuration. You can also understand what is the module of message when reading the log.